redhat-appstudio/service-provider-integration-operator
# Add requests/limits for ancillary containers (kube-rbac-proxy) and correct main service limits

Signed-off-by: Max Shaposhnyk <mshaposh@redhat.com>

### What does this PR do?
- Add requests/limits for ancillary containers (kube-rbac-proxy)
 - Reduced manager and oauth limits a little, in correspondence with actual consumption (see graphs from staging below. They show spi-manager (or spi-oauth on 3rd image) memory, kube-rbac memory, and their sum )

### Screenshot/screencast of this PR

![image](https://user-images.githubusercontent.com/1651062/215460182-a6aa7aea-d1e2-4b25-8fc3-387e5c95cb60.png)
![image](https://user-images.githubusercontent.com/1651062/215509608-d624112a-7504-4f68-968d-ebbfa57a6b2e.png)
![image](https://user-images.githubusercontent.com/1651062/215509633-0385002a-c8f3-4e25-83a0-9d5eb5b4ab26.png)


### What issues does this PR fix or reference?

https://issues.redhat.com/browse/SVPI-313

### How to test this PR?

 Deploy and verify containers limits are set

```
quay.io/redhat-appstudio/pull-request-builds:spi-operator-#
quay.io/redhat-appstudio/pull-request-builds:spi-oauth-#
```
